apis/Google.Cloud.PubSub.V1/Google.Cloud.PubSub.V1/Tasks/TaskHelper.cs

```diff
@@ -107,46 +109,49 @@ public static TaskAwaitable<T> ConfigureAwaitWithFinally<T>(this TaskHelper task
             return taskHelper.ConfigureAwait(Inner());
         }
 
-        public static TaskAwaitable<bool> ConfigureAwaitHideCancellation(this TaskHelper taskHelper, Task task)
+        public static TaskAwaitable<bool> ConfigureAwaitHideCancellation(this TaskHelper taskHelper,
```

Comment:
Given the number of places you call this with just a simple task, perhaps it should be overloaded?

Reply:
Could do, but having a method that takes a Task makes it easy to call it by mistake when the correct code should call this method.